content/desktop/install/msi/install-and-configure.md

```diff
@@ -0,0 +1,149 @@
+
+    - Disable Windows Container usage
+
+    - Select the engine for Docker Desktop. Either WSL or Hyper-V. If your system only supports one of the two options, you will not be able to select which backend to use.
+
+5. Follow the instructions on the installation wizard to authorize the installer and proceed with the install.
+
+6. When the installation is successful, select **Finish** to complete the installation process.
+
+If your admin account is different to your user account, you must add the user to the **docker-users** group:
+1. Run **Computer Management** as an **administrator**.
+2. Navigate to **Local Users and Groups** > **Groups** > **docker-users**. 
+3. Right-click to add the user to the group.
+4. Sign out and sign back in for the changes to take effect.
+
+## Install from the command line
+
+This section covers command line installations of Docker Desktop using PowerShell. It provides common installation commands that you can run. You can also add additional arguments which are outlined in [configuration options](#configuration-options).
+
+When installing Docker Desktop, you can choose between interactive or non-interactive installations. 
+
+Interactive installations, without specifying `/quiet` or `/qn`, display the user interface and let you select your own properties. 
+
+When installing via the user interface it is possible to:
+
+- Choose the destination folder
+- Create a desktop shortcut
+- Configure the Docker Desktop service startup type
+- Disable Windows Containers
+- Choose between the WSL or Hyper-V engine
+
+Non-interactive installations are silent and any additional configuration must be passed as arguments.
+
+### Common installation commands
+
+> **Important**
+>
+> Admin rights are required to run any of the following commands.
+{ .important }
+
+#### Installing interactively with verbose logging
+
+```powershell
+msiexec /i "DockerDesktop.msi" /L*V ".\msi.log"
+```
+
+#### Installing interactively without verbose logging
+
+```powershell
+msiexec /i "DockerDesktop.msi"
+```
+
+#### Installing non-interactively with verbose logging
+
+```powershell
+msiexec /i "DockerDesktop.msi" /L*V ".\msi.log" /quiet
+```
+
+#### Installing non-interactively and suppressing reboots
+
+```powershell
+msiexec /i "DockerDesktop.msi" /L*V ".\msi.log" /quiet /norestart
+```
+
+#### Installing non-interactively with Admin settings
+
+```powershell
+msiexec /i "DockerDesktop.msi" /L*V ".\msi.log" /quiet /norestart ADMINSETTINGS="{"configurationFileVersion":2,"enhancedContainerIsolation":{"value":true,"locked":false}}" ALLOWEDORG="docker.com"
+```
+
+> **Tip**
+>
+> Some useful tips to remember when creating a value that expects a JSON string as it’s value:
+> 
+> - The property expects a JSON formatted string
+> - The string should be wrapped in double quotes
+> - The string should not contain any whitespace
+> - Property names are expected to be in double quotes
+{ .tip }
+
```

Comment:
New section?

## Installing with the passive display option

When installing the MSI you can use the [`/passive` display option](https://learn.microsoft.com/en-us/windows/win32/msi/standard-installer-command-line-options#passive). This would typically replace `/quiet` in scenarios where you want to perform a non-interactive installation but show a progress dialog.

In passive mode the installer won't display any prompts or error messages to the user and the installation cannot be cancelled.

For example;

```powershell
msiexec /i "DockerDesktop.msi" /L*V ".\msi.log" /passive /norestart
```